Integrations/integration-Intezer.yml

```diff
@@ -232,25 +235,36 @@ script:
             }
             md += 'Verdict: **' + data.verdict + '** (' + data.sub_verdict + ')\n';
             md += '[' + 'Analysis Link' + '](' + data.analysis_url + ')\n';
-
             // set return entry with context
-            entries.push({
+            entry = {
               Type: entryTypes.note,
               Contents: res.body,
               ContentsFormat: formats.json,
               HumanReadable: md,
+              ReadableContentsFormat: formats.markdown,
               EntryContext: ec
-            });
+            };
         }
-        return entries;
+        return entry;
     };
 
     switch (command) {
         case 'test-module':
             // send is available
             return sendRequest('GET', IS_AVAILABLE_URL).obj.is_available;
+
         case 'file':
-            return analyzeSHA256(args.file, args.codeItuemType, args.maxRetries, args.delay);
+            if ((args.sha256 && args.file) || (!args.sha256 && !args.file)) {
+                throw 'Specify only 1 of the arguments: file, sha256';
```

Comment:
return error entry or catch the exception and return error entry and to the log. we dont want to print stack to the warroom